### DISCUSSION TITLE
Code search by repository doesn't work

### DISCUSSION THREAD

**Discussion by nikpivkin** [community/community#138767] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Example url: https://github.com/search?q=repo%3Alocalstack%2Flocalstack+CreateVolumeRequest&type=code

The search returns no results, but the repository contains a [CreateVolumeRequest](https://github.com/localstack/localstack/blob/master/localstack-core/localstack/aws/api/ec2/__init__.py#L8765) class.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @nikpivkin , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by look** [comment, marked as answer]
Sorry for the inconvenience @nikpivkin. The reason this does not match is that the file with the match is 806 KB, which is larger than the [maximum file size of 350 KB](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations).
